src/theme/fonts/fonts.css

```
@@ -96,6 +96,6 @@
   font-family: 'Source Code Pro';
   font-style: normal;
   font-weight: 500;
-  src: local('Source Code Pro Medium'), local('SourceCodePro-Medium'),
-       url('source-code-pro-v11-all-charsets-500.woff2') format('woff2');
+  src: url('source-code-pro-v11-all-charsets-500.woff2') format('woff2'),
+       local('Source Code Pro Medium'), local('SourceCodePro-Medium');
```

It may make sense to not even try the local version of SCP, which would also match Rustdoc's behavior: https://github.com/rust-lang/rust/blob/7fc048f0712ba515ca11fac204921b9ad8a0c5a3/src/librustdoc/html/static/rustdoc.css#L36-L43

```suggestion
  src: url('source-code-pro-v11-all-charsets-500.woff2') format('woff2');
```

---

It would perhaps be better to fall back to another font where syntax highlighting *does* work, rather than using a potentially buggy local version of SCP where syntax highlighting does not work.

---

Also see @nrc's comment here: https://github.com/rust-lang/rust/issues/24355#issuecomment-248443541

---

I think it would be fine to completely remove the `local` setting here.  I believe they are processed in order, and would only use local if the first one doesn't load for some reason, which should be rare (and rarer still that someone has source code pro installed locally).

---

Well I'm guessing a decent number of programmers have SCP installed, but I agree that it's unlikely that the bundled version wouldn't load and yet the page would. Also it would just fall back to a different monospace font which is fine:

https://github.com/rust-lang/mdBook/blob/4b773024ae9872c80b7e866e03472b08523e33b3/src/theme/css/general.css#L23-L26